ui/src/composables/use-cached-filters.js

```diff
@@ -0,0 +1,74 @@
+import useContext from './use-context'
+import { SessionStorage } from 'quasar'
+import { reactive } from 'vue'
+import { useRouter, useRoute } from 'vue-router'
+
+const cachedFilters = reactive({})
```

Comment:
To pensando se precisa ser um `reactive` pois em nenhum momento utilizamos essa chave em um template ou dentro de uma computed, watch, etc...